common/math/geometry/src/solver/polynomial_solver.cpp

```diff
@@ -105,15 +115,14 @@ std::vector<double> PolynomialSolver::solveCubicEquation(
 int PolynomialSolver::solveP3(std::vector<double> & x, double a, double b, double c) const
 {
   x = std::vector<double>(3);
-  const double eps = std::numeric_limits<double>::epsilon();
   double a2 = a * a;
   double q = (a2 - 3 * b) / 9;
   double r = (a * (2 * a2 - 9 * b) + 27 * c) / 54;
   // equation x^3 + q*x + r = 0
   double r2 = r * r;
   double q3 = q * q * q;
   double A, B;
-  if (r2 <= (q3 + eps)) {  //<<-- FIXED!
+  if (r2 <= (q3 + tolerance)) {  //<<-- FIXED!
```

Comment:
What does the comment `<<-- FIXED!` mean? If you cannot explain a rational reason for this comment to exist, please delete it.

Reply:
Sorry, I deleted this comment because I can only assume that the comment was somehow left behind because its intent was unclear.

https://github.com/tier4/scenario_simulator_v2/pull/996/commits/6f96099d56e2a85b309793f48376872f14acb461

Reply:
I confirmed it was comes from We deleted comments that were unclear in intent, reviewed the process, and added comments to sections where the process was unclear.